libs/ui/src/SummaryList/SummaryList.tsx

```diff
@@ -0,0 +1,25 @@
+import { forwardRef } from 'react'
```

Comment:
To put this in the ui library I think this is an unnecessary abstraction since we're not going to reuse this...